src/task-lists-element.js

```diff
@@ -14,8 +14,7 @@ export default class TaskListsElement extends HTMLElement {
       if (!checkbox.classList.contains('task-list-item-checkbox')) return
 
       this.dispatchEvent(
-        new CustomEvent('task-lists:check', {
-          bubbles: true,
```

Comment:
I think a bubbling rule could be to bubble for a custom element that cannot contain itself as a child. That will let us use delegated event listeners and directly-bound listeners.

When an element can contain itself as a child, like `<details>`, it makes sense to not bubble to avoid accidentally targeting the wrong element in a delegated event listener.